packages/hunspell.rb

```diff
@@ -37,14 +31,18 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+
+  def self.postinstall
     system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
     system "install -Dm644 dictionaries/en/en_US.aff #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.aff"
     system "install -Dm644 dictionaries/en/en_US.dic #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.dic"
     system "install -Dm644 dictionaries/en/en_US.aff $HOME/Library/Spelling/en_US.aff"
     system "install -Dm644 dictionaries/en/en_US.dic $HOME/Library/Spelling/en_US.dic"
```

Comment:
It seems more logical to me to set these steps in postinstall. Thus one could run postinstall to update dictionaries regularly.

Reply:
These lines aren't needed, actually, and can be removed.